insights/specs/__init__.py

```diff
@@ -830,6 +830,7 @@ class Specs(SpecSet):
     vsftpd_conf = RegistryPoint(filterable=True)
     watchdog_logs = RegistryPoint(filterable=True, multi_output=True)
     wc_proc_1_mountinfo = RegistryPoint()
+    wc_var_lib_pcp_config_pmda = RegistryPoint(no_obfuscate=['hostname', 'ip'])
```

Comment:
It makes me think adding a new datasource with filterable feature, so it's easy for the following rules to `add_filter` for the required `wc` target files

`wc_files`